src/pages/index.js

```diff
@@ -1,91 +1,180 @@
 import React from 'react';
-import { Redirect } from 'react-router';
 import clsx from 'clsx';
 import Layout from '@theme/Layout';
 import Link from '@docusaurus/Link';
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 import styles from './styles.module.css';
+import { ResponsivePlayer } from '../components'
 
 const features = [
   {
-    title: <>Easy to Use</>,
-    imageUrl: 'img/undraw_docusaurus_mountain.svg',
+    title: <>Server management</>,
```

Comment:
suggestion (take it or leave it) - perhaps focus more on persona?

```suggestion
    title: <>Server Admins</>,
```

Reply:
Good point, but lets keep it focused to the topic, and mention the persona in the description.